subworkflows/nf-core/utils_nfcore_pipeline/main.nf

```diff
@@ -267,11 +270,11 @@ def imNotification(summary_params) {
     def misc_fields = [:]
     misc_fields['start']                                = workflow.start
     misc_fields['complete']                             = workflow.complete
-    misc_fields['scriptfile']                           = workflow.scriptFile
+    misc_fields['scriptfile']                            = workflow.scriptFile
     misc_fields['scriptid']                             = workflow.scriptId
-    if (workflow.repository) misc_fields['repository']  = workflow.repository
-    if (workflow.commitId)   misc_fields['commitid']    = workflow.commitId
-    if (workflow.revision)   misc_fields['revision']    = workflow.revision
+    if (workflow.repository) misc_fields['repository']   = workflow.repository
+    if (workflow.commitId)   misc_fields['commitid']     = workflow.commitId
+    if (workflow.revision)   misc_fields['revision']     = workflow.revision
```

Comment:
unnecessary whitespace changes

Reply:
I'm guessing theres a local display issue in your editor- perhaps a soft wrap or something like that? if you look at these lines [in github](https://github.com/nf-core/modules/blob/511d039d5ae004ddbd94cfe05c107ab29d8047c5/subworkflows/nf-core/utils_nfcore_pipeline/main.nf#L270-L280) the added spaces bring these out of alignment.